File: xenon-android-library/src/main/kotlin/com/abubusoft/xenon/XenonStartupTask.java
/**
 * 
 */
package com.abubusoft.xenon;

import com.abubusoft.xenon.android.XenonActivitySplash;

/**
 * <p>
 * Contiene il task da eseguire durante la visualizzazione dello splash screen.
 * </p>
 * 
 * @author Cesco
 * 
 */
public interface XenonStartupTask {

	/**
	 * <p>
	 * effettua il task.
	 * </p>
	 * 
	 * @param splashActivity
	 */
	void doTask(XenonActivitySplash splashActivity);

}

File: xenon-android-library/src/main/kotlin/com/abubusoft/xenon/XenonStartupTask.kt
/**
 *
 */
package com.abubusoft.xenon

import com.abubusoft.xenon.android.XenonActivitySplash

/**
 *
 *
 * Contiene il task da eseguire durante la visualizzazione dello splash screen.
 *
 *
 * @author Cesco
 */
interface XenonStartupTask {
    /**
     *
     *
     * effettua il task.
     *
     *
     * @param splashActivity
     */
    fun doTask(splashActivity: XenonActivitySplash?)
}